controllers/argocd/status.go

```diff
@@ -250,16 +252,33 @@ func (r *ReconcileArgoCD) reconcileStatusNotifications(cr *argoprojv1a1.ArgoCD)
 func (r *ReconcileArgoCD) reconcileStatusHost(cr *argoprojv1a1.ArgoCD) error {
 	cr.Status.Host = ""
 	cr.Status.Phase = "Available"
-	if cr.Spec.Server.Route.Enabled {
-		if !IsRouteAPIAvailable() {
-			log.Info("Routes not available in non-OpenShift environments, please use Ingresses instead")
-			return nil
-		}
```

Comment:
If it is in the original code, we should log preserve message. The fix has the side effect of deleting the log message.

Reply:
I think it should be fine to remove this, since it should be common knowledge that routes are only available on openshift and not kubernetes, at least amongst the users of this operator